src/admin/MenuSetAdmin.php

```diff
@@ -17,6 +17,8 @@
  */
 class MenuSetAdmin extends ModelAdmin
 {
+    const CMS_ACCESS_PERMISSION = 'CMS_ACCESS_' . self::class;
```

Comment:
For every sub-class of `LeftAndMain`, SilverStripe automatically generates a permission called `CMS_ACCESS_[fully qualified classname]`. If a user does not have this permission, they cannot see that admin panel in the CMS menu.

I created this constant to use it in the `MenuLink::canView` and `MenuLink::canCreate` methods, rather than hardcoding a magic value in there directly.